File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Python/questao08.py
Vf = float(input('Qual o valor final que você deseja obter:  '))
i = float(input('Qual vai ser a rentabilidade: '))
i = i / 100
meses = int(input('Quantos meses? '))
vi = Vf / (i+1)**meses
print('Valor inicial {:.2f}'.format(vi))

File: 01_Basico/01 - Conceitos_Basicos/Respostas_em_Kotlin/questao08.kt
fun main() {
    println("Qual valor final que você deseja obter: ")
    var vf = readLine()!!.toDouble()
    println("Qual vai ser a rentabilidade: ")
    var i = readLine()!!.toInt()
    var j = i / 100
    var m: Double = 1.0 + j
    println("Quantos meses ?")
    var meses = readLine()!!.toInt()
    var vi = vf / Math.pow(m.toDouble(), meses.toDouble())
   
    println(vi)
}